MSCI445-Lab2/EmailLab/ErrorWindow.xaml.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+
+namespace EmailLab
+{
+    public partial class ErrorWindow : Window, INotifyPropertyChanged
+    {
+        String errors;
+
+        public ErrorWindow(String errors)
+        {
+            InitializeComponent();
+            this.errors = errors;
+            DataContext = this;
+            this._errormessage = this.errors;
+        }
+
+        private string _errormessage = "this is the error message";
```

Comment:
can we move all class variables to the very top? i.e. just like how line 23 you have `string errors`